libs/extensions/std/exec/src/gtfs-rt-interpreter-executor.ts

```diff
@@ -51,37 +51,28 @@ export class GtfsRTInterpreterExecutor
   }
 
   private async parseFeedMessage(
-    entity_type: string,
+    entityType: string,
     feedMessage: GtfsRealtimeBindings.transit_realtime.FeedMessage,
     context: ExecutionContext,
   ): Promise<Either<Error, Sheet>> {
-    return new Promise((resolve) => {
-      switch (entity_type) {
-        case 'trip_update':
-          // Extract the trip updates from thee feed message
-          resolve(this.parseTripUpdates(feedMessage, context));
-          break;
-        case 'alert':
-          // Extract the alerts from thee feed message
-          resolve(this.parseAlerts(feedMessage, context));
-          break;
-        case 'vehicle':
-          // Extract vehicle positions from thee feed message
-          resolve(this.parseVehiclePositions(feedMessage, context));
-          break;
-
-        // Entity invalid
-        default:
-          resolve(
-            E.left(
-              new Error(
-                `Entity should be either "trip_update", "alert" or "vehicle". Please check the argument "entity" of the GtfsRTInterpreterblock.`,
-              ),
-            ),
-          );
-          break;
-      }
-    });
+    switch (entityType) {
+      case 'trip_update':
+        // Extract the trip updates from thee feed message
+        return this.parseTripUpdates(feedMessage, context);
+      case 'alert':
+        // Extract the alerts from thee feed message
+        return this.parseAlerts(feedMessage, context);
+      case 'vehicle':
+        // Extract vehicle positions from thee feed message
+        return this.parseVehiclePositions(feedMessage, context);
+      // Entity invalid
+      default:
+        return E.left(
+          new Error(
+            `Entity should be either "trip_update", "alert" or "vehicle". Please check the argument "entity" of the GtfsRTInterpreterblock.`,
```

Comment:
Please include the argument (`entityType`). A better error message would be `Entity ${entityType} not allowed for block GtfsRTInterpreterblock, expected "trip_update", "alert" or "vehicle".`

Reply:
Thanks, changed the errormessage. As far as i understood it correct, with my new validation function the default case of this switch statement should actually never be executet. 

`l[ibs/extensions/std/lang/src/gtfs-rt-interpreter-meta-inf.ts](https://github.com/jvalue/jayvee/blob/bc25b49c166ff0defe05be2b45cfe44d1a373830/libs/extensions/std/lang/src/gtfs-rt-interpreter-meta-inf.ts#L108)`